index.js

```diff
@@ -157,7 +168,7 @@ class PluginHttp extends EventEmitter {
       ? await this._generateUrl(data)
       : this._url
 
-    const res = await fetch(url, {
+    const res = await this._fetch(url, {
       method: 'POST',
       body: data,
       headers: {
```

Comment:
It requires the plugin to parse the ILP packet, which it currently doesn't do (unless it's a multilateral plugin). I don't want to add that inefficiency in for all packets, but it might be worth adding it as a flag in case any other implementations expect it to be set.